### PR TITLE
Adds more interesting cricket bat desc

### PR DESCRIPTION
**Pull request by Bawhoppen**
Requested by @RaveRadbury

### CODE REVIEW

**Comment by Wallemations** [comment]
What's this a reference to?

**Comment by Bawhoppen** [comment]
@Wallemations https://www.youtube.com/watch?v=Oc_CesMUb5E

**Comment by Wallemations** [comment]
> @Wallemations https://www.youtube.com/watch?v=Oc_CesMUb5E

I'm still partial to my original reference, though I admit I'm just a big fan of Shaun of the Dead so I might be biased.
If a majority of people believe this is a better reference than the original, then go ahead.

**Comment by Bawhoppen** [comment]
Ah I didn't realize it was a reference or you added it. No worries, it was mostly a joke for Rave's sake during a stream. I'll let the maintainers decide what they think is funnier/more popular.

**Comment by Wallemations** [comment]
> Ah I didn't realize it was a reference or you added it. No worries, it was mostly a joke for Rave's sake during a stream. I'll let the maintainers decide what they think is funnier/more popular.

All good, for posterity [here's what I was originally referencing](https://youtu.be/T1GYsCMCLpo) when I made the cricket bat.

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -663,7 +663,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	. = ..()
 	if(prob(1))
 		name = "cricket bat"
-		desc = "You've got red on you."
```
Comment (coiax) [inline code/game/objects/items/weaponry.dm:666]: Use prob(50) to select the old or the new desc. Let's just double down on code branching, I bet it'll make someone cry in the future.

**Review by coiax — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (Bawhoppen) [reply code/game/objects/items/weaponry.dm:666]: now this is a brilliant idea

**Pull request merged**